dev/com.ibm.ws.rest.handler.validator_fat/fat/src/com/ibm/ws/rest/handler/validator/fat/ValidateDataSourceTest.java

```diff
@@ -304,22 +304,6 @@ public void testMultiple() throws Exception {
         assertTrue(err, stack.getString(0).startsWith("org.apache.derby."));
         assertTrue(err, stack.getString(1).startsWith("org.apache.derby."));
         assertTrue(err, stack.getString(2).startsWith("org.apache.derby."));
-
-        // [5]: config.displayId=transaction/dataSource[default-0]
```

Comment:
Same comment as previously. Do not remove the only tests of dataSource nested under singleton pid.